### DISCUSSION TITLE
Linux Mint Menu Launcher - Wrong Location

### DISCUSSION THREAD

**Discussion by tmanok** [congard/nvidia-system-monitor-qt#18] [General]
Hey everyone, rather than creating an issue, I wanted to talk about this first to see if I had done something wrong. 

While first looking at this project I was in a bit of a rush. While in a rush, I followed this older guide: https://www.linuxuprising.com/2019/06/2-tools-for-monitoring-nvidia-gpus-on.html to install it. That was fine and everything built nicely, however it didn't give me a desktop shortcut or a menu option. 

Today, checking out the GitHub ReadMe I noticed that there are pre-compiled .deb installers, so I tried one out. Only issue I noticed was that the menu item launches /usr/bin/qnvsm when the location that the executable lives in is /usr/local/bin/qnvsm so without making manual changes to the menu item, nothing launches.

I'm wondering if my initial install following that guide is what caused the issue or if my desktop manager might be the issue. Perhaps the GNOME "9 dots" or "applications menu" is quite different from the cinnamon menu, or maybe the .deb is at fault. 

System: Fresh install of Linux Mint 20.2 running Cinnamon. 

Thanks!

**Comment by congard** [comment]
Hi, thank you for opening this discussion. Could you please post the output of the command below?
`which qnvsm && ls /usr/bin/qnvsm && ls /usr/local/bin/qnvsm`
